### DISCUSSION TITLE
Can't Hear Audio In Tutorial After Slider Gameplay

### DISCUSSION THREAD

**Discussion by jessiejs** [ppy/osu#20677] [Q&A]
### Type

Game behaviour

### Bug description

In new beginnings, by pishifat, the tutorial, unlike osu!stable, after the slider part of the song the voice stops. Not the music though.

### Version

2022.1008.2-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9740649/database.log)
[network.log](https://github.com/ppy/osu/files/9740650/networ
[updater.log](https://github.com/ppy/osu/files/9740653/updater.log)
k.log)
[performance.log](https://github.com/ppy/osu/files/9740651/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9740652/runtime.log)

**Comment by bdach** [comment]
Cannot reproduce, it plays just fine here. Please attach a video of this.

**Comment by jessiejs** [comment]
OBS is just recording a black screen. I've tried Game Capture and Display Capture.

**Comment by jessiejs** [comment]
https://github.com/CodeLikeCrazE/cdn/blob/main/2022-10-11%2017-30-44.mkv?raw=true

**Reply by ghost** [reply]
(the video file seems to have audio turned off. maybe try re-recording?)

**Comment by jessiejs** [comment]
Odd. This doesn't happen on dev builds.

**Reply by bdach** [reply]
I can't reproduce either on dev builds or on latest release.
